Sources/Public/CalendarView.swift

```diff
@@ -388,6 +400,23 @@ public final class CalendarView: UIView {
 
   fileprivate var previousPageIndex: Int?
 
+  fileprivate lazy var scrollView: NoContentInsetAdjustmentScrollView = {
+    let scrollView = NoContentInsetAdjustmentScrollView()
+    scrollView.showsVerticalScrollIndicator = false
+    scrollView.showsHorizontalScrollIndicator = false
+    scrollView.delegate = scrollViewDelegate
+    return scrollView
+  }()
+
+  fileprivate lazy var multipleDaySelectionGestureRecognizer: UILongPressGestureRecognizer = {
+    let gestureRecognizer = UILongPressGestureRecognizer(
+      target: self,
+      action: #selector(multipleDaySelectionGestureRecognized(_:)))
+    gestureRecognizer.allowableMovement = .greatestFiniteMagnitude
```

Comment:
I've seen this value confuse CoreText in the past and ended up using `1_000_000`.  I'd double check the console log to make sure, but if that's clear you should be good since it's working.

Reply:
No errors / warnings and works correctly, so I think we're safe here